minecraft-ftb-fabric/minecraft-ftb-fabric.json

```diff
@@ -0,0 +1,133 @@
+{
+  "name": "minecraft-ftb-fabric",
+  "display": "FTB - Fabric",
+  "type": "minecraft-java",
+  "install": [
+    {
+      "type": "javadl",
+      "version": "${javaversion}"
+    },
+    {
+      "files": [
+        "https://api.modpacks.ch/public/modpack/${modpack_id}/${modpack_version}/server/linux"
+      ],
+      "type": "download"
+    },
+    {
+      "commands": [
+        "chmod u+x linux",
+        "./linux ${modpack_id} ${modpack_version} --auto --noscript"
+      ],
+      "type": "command"
+    },
+    {
+      "source": "fabric-*.jar",
+      "target": "FabricServer.jar",
+      "type": "move"
+    },
+    {
+      "target": "eula.txt",
+      "text": "eula=${eula}",
+      "type": "writefile"
+    },
+    {
+      "target": "server.properties",
+      "text": "server-ip=${ip}\nserver-port=${port}\nmotd=${motd}\n",
+      "type": "writefile"
+    }
+  ],
+  "run": {
+    "stop": "stop",
+    "command": "java${javaversion} -server ${args} -Dlog4j2.formatMsgNoLookups=true -Xmx${memory}M -Xms${memory}M -Dterminal.jline=false -Dterminal.ansi=true -jar FabricServer.jar nogui",
+    "workingDirectory": "",
+    "pre": [],
+    "post": [],
+    "environmentVars": {}
+  },
+  "data": {
+    "args": {
+      "type": "option",
+      "display": "Java Arguments",
+      "required": true,
+      "options": [
+        {
+          "value": "-XX:+UnlockExperimentalVMOptions -XX:+UseShenandoahGC -XX:+AlwaysPreTouch",
+          "display": "ShenandoahGC"
+        },
+        {
+          "value": "-XX:+UnlockExperimentalVMOptions -XX:+UseG1GC",
+          "display": "G1GC"
+        }
+      ]
+    },
+    "eula": {
+      "type": "boolean",
+      "desc": "Do you (or the server owner) agree to the <a href='https://account.mojang.com/documents/minecraft_eula'>Minecraft EULA?</a>",
+      "display": "EULA Agreement",
+      "required": true,
+      "value": "false"
+    },
+    "ip": {
+      "type": "string",
+      "desc": "What IP to bind the server to",
+      "display": "IP",
+      "required": true,
+      "value": "0.0.0.0"
+    },
+    "javaversion": {
```

Comment:
Watch the version here, because 8 doesn't work for new versions. I'd probably say assume newer packs, so a newer version of Java so it doesn't "break" immediately.

(I wish APIs had the versions better....)

Reply:
Changed default to 17